FrontEnd/src/main/java/resourceDisplay/AssetsRepository.java

```diff
@@ -237,11 +237,18 @@ public TableQueryObject createTableQueryObject(DatabaseClient dbClient, String f
         }
         String columns = String.join(", ", columnNames);
         String statementString;
-        if (filterType != null) {
-            statementString = String.format("SELECT %s FROM Main_Assets WHERE %s = @%s"
-                    , columns, filterType, filterType);
+        if (filters.isEmpty()) {
+            statementString = String.format("SELECT %s FROM Main_Assets ORDER By %s", columns,
+                    columnNames.get(0));
         } else {
-            statementString = String.format("SELECT %s FROM Main_Assets", columns);
+            List<String> forWhere = new ArrayList<>();
```

Comment:
a name like `queryFilters` would explain better what you're collecting here

Reply:
done